ncl/receipts/climatology_functions.ipynb

```diff
@@ -0,0 +1,212 @@
+{
```

Comment:
<p>On the site it looks like it is currently throwing <code>AssertionError</code> for <code>calcMonAnomTLL</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Yeah, I knew this was happening and meant to mention something in the PR description (so thanks for raising this!).  

I'm fairly confident it's a difference in types and associated handling that's causing the diff to be just over the threshold here.  

I'm somewhat inclined to just change the threshold since I think we decided this type of thing was beyond the scope of what we wanted to discuss in these, but wanted to check in with others.

I'll take another look as I go through and update some of these other items.

Reply:
So it looks like the data for these calculations is stored in the file as float / single precision and NCL is retaining that type through the calculations whereas Python is not.  

The numbers are about as close as I would expect given the circumstances.  

I'm going to adjust to `rel_tol=1e-05`, which I think is reasonable if we're not looking to dig into these precision / types differences.

```
clmMonTLL: 
	python:		0.8063018918037415
	ncl:		0.8063018

calcDayAnomTLL: 
	python:		-0.02774369716644287
	ncl:		-0.02774364
```

Reply:
Sounds reasonable to me.